src/components/SimilarPosts/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Row, Container } from "react-bootstrap";
+import Post from "../Post/index";
+import "./index.css";
+
+export default function SimilarPosts({ similarPostData }) {
+  console.log(similarPostData);
+  return (
+    <Container className="similarPostContent">
+      <h1 className="text-center title">Similar Posts</h1>
+      <Row className="justify-content-md-center">
+        {similarPostData
```

Comment:
You can use slice here like this
`similarPostData
           .slice(0,3)
           .map((data) => {
             return <Post postInfo={data} />;
           }) `